Added 3d printed case to documentation

bob9 opened the pull request:
Updated Readme with link to the STL on [printables](https://www.printables.com/model/762529-esp32-wroom-32u-casing). 

Added example image of printed STL.

bob9 committed:
Added 3d printed case to documentation

bob9 committed:
Fix typo

klaasnicolaas commented on README.md:
```diff
@@ -210,4 +210,14 @@ Row to show lap or gap time
\ No newline at end of file
+> Rows 10-14 are used by `Show Post-Race Results` when it is enabled. You can use these rows if the feature is disabled.
+
+
+### 3D Printed Case
+Available on Printables
+https://www.printables.com/model/762529-esp32-wroom-32u-casing
+
+
+![Case](docs/3DPrint/esp_case.jpg)
+
+
```
```suggestion
### 3D Printed Case

Available on Printables
https://www.printables.com/model/762529-esp32-wroom-32u-casing

![Case](docs/3DPrint/esp_case.jpg)

```

klaasnicolaas reviewed (changes requested):
Lets remove the extra black lines and follow the style guide for markdown. Would also place this much higher, perhaps below requirements.

./Klaas

gp-arconomy committed:
Change to more like other styles

bob9 commented:
I have updated the readme and removed black lines and changed the header to
be consistent with other headers. There was a guy who commented on the
discord that the stl from printable would be good if it was on the
documentation so thought i'd add it.

Cheers,

bob9

On Tue, May 28, 2024 at 11:18 PM Klaas Schoute ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
>
> Lets remove the extra black lines and follow the style guide for markdown.
> ------------------------------
>
> In README.md
> <https://github.com/i-am-grub/VRxC_ELRS/pull/2#discussion_r1617230062>:
>
> > +
> +### 3D Printed Case
> +Available on Printables
> +https://www.printables.com/model/762529-esp32-wroom-32u-casing
> +
> +
> +![Case](docs/3DPrint/esp_case.jpg)
> +
> +
>
> ⬇️ Suggested change
>
> -
> -### 3D Printed Case
> -Available on Printables
> -https://www.printables.com/model/762529-esp32-wroom-32u-casing
> -
> -
> -![Case](docs/3DPrint/esp_case.jpg)
> -
> -
> +### 3D Printed Case
> +
> +Available on Printables
> +https://www.printables.com/model/762529-esp32-wroom-32u-casing
> +
> +![Case](docs/3DPrint/esp_case.jpg)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/i-am-grub/VRxC_ELRS/pull/2#pullrequestreview-2082765239>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AANMX7O2T4J3NM3EFHTGJHTZER7YTAVCNFSM6AAAAABIMZCVCWVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDAOBSG43DKMRTHE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

klaasnicolaas commented:
> There was a guy who commented on the discord

That's me 😅

bob9 commented:
Your welcome. Readme updated. :)

On Wed, 29 May 2024, 4:42 pm Klaas Schoute, ***@***.***>
wrote:

> There was a guy who commented on the discord
>
> That's me 😅
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/i-am-grub/VRxC_ELRS/pull/2#issuecomment-2136653039>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AANMX7LSG2L4CDMXKO6YUULZEV2GXAVCNFSM6AAAAABIMZCVCWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDCMZWGY2TGMBTHE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
